final/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.8"
+services:
+  bitcoin-core:
+    build:
+      dockerfile: ./bitcoind/Dockerfile
+    image: bitcoind
+    container_name: bitcoind
+    command:
+      - --testnet
+      - -pid=/home/bitcoin/.bitcoin/testnet3/bitcoind-testnet.pid
+    volumes:
+      - data:/home/bitcoin/.bitcoin
+    expose:
+      - 8332
+      - 8333
+      - 18332
+      - 18333
+      - 18444
+    ports:
+      - 8333:8333
+      - 18333:18333
+      - 38333:38333
+    networks:
+      LN_testing:
+        aliases:
+          - bitcoind
+
+  core-lightning:
+    build:
+      dockerfile: ./lightningd/Dockerfile
+    image: lightningd
+    container_name: lightningd
+    command:
+      - --conf=/home/lightning/.lightning/testnet-config
+      - --bitcoin-rpcconnect=bitcoind
+      - --bind-addr=core-lightning:19735
+      - --announce-addr=core-lightning:19735
+    volumes:
+      - clightning:/home/lightning/.lightning
+    expose:
+      - 9735
+      - 9835
+    ports:
+      - 9735:9735
```

Comment:
why connect these to host?

Reply:
I was referencing this [repo](https://github.com/LukasBahrenberg/lightning-dockercompose/blob/master/docker-compose.yaml) and they say its for the lightning peers not really sure how does that work.

Reply:
ah, yes, that's the p2p port